### DISCUSSION TITLE
가이드에 있는 EventTrigger 예제 그대로 했는데 안되네요.

### DISCUSSION THREAD

**Discussion by pda017** [naverz/zepeto-studio-kor#287] [5. Q&A]
![image](https://user-images.githubusercontent.com/77001589/151410925-a6ffbdb4-5738-4c2f-8aee-0e7c153d1ff2.png)

Property 'Add' does not exist on type 'List$1<any>'. 

List$1 클래스에 Add가 없다고 에러가 뜹니다. 무시하고 실행할려니 컴파일이 안된다고 에러뜨네요.

이전 게시글에 답이 있었네요. 해결했습니다.

**Comment by hyeonjin-jo** [comment]
안녕하세요.

해결되셔서 다행입니다

완료 처리하겠습니다.
